### ISSUE TITLE
Communication only works in one direction

### ISSUE THREAD

**Issue by JustinDrake** [duosearch/openbazaar.chat#8]
(Reported by @hoffmabc on Slack.) Communications work from openbazaar.chat to openbazaar-desktop, but not the other way round.

The error on the reference client is 

```
{
   “success”: false,
   “reason”: “routing: not found”
}
```

![pasted_image_at_2017_04_21_09_20](https://cloud.githubusercontent.com/assets/731743/25303692/2da232a2-2750-11e7-914f-cd974e6e2edc.png)

**Comment by jackkleeman** [comment]
I suspect this is a public key publishing issue

**Comment by JustinDrake** [comment]
There might be several issues. I thought that openbazaar-go now persists open connections so the public key shouldn't be required (unless the connection was cut somehow).

**Comment by jackkleeman** [comment]
Yes it should be. Probably best to work out where the log is coming from

**Comment by jackkleeman** [comment]
https://github.com/OpenBazaar/openbazaar-go/blob/5ada9056398ced7f6bc825dc7ae2c148692128dc/vendor/gx/ipfs/QmUc6twRJRE9MNrUGd8eo9WjHHxebGppdZfptGCASkR7fF/go-libp2p-routing/routing.go

It does indeed seem like a public key issue. This implies that the online message failed to send.
